modules/cas_cache/classifier.c

```diff
@@ -686,7 +686,7 @@ static cas_cls_eval_t _cas_cls_file_offset_test(
 	if (!dentry)
 		return cas_cls_eval_no;
 
-	offset = PAGE_SIZE * io->page->index +
+	offset = PAGE_SIZE * cas_page_index(io->page) +
```

Comment:
Aren't there any macros available to convert page to index? I looked for it but didn't found any... but found this instead: https://elixir.bootlin.com/linux/v6.16.1/source/include/linux/pagemap.h#L1008
Maybe we can get the offset with this?